packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -203,7 +207,9 @@ export class SignUp extends BaseResource implements SignUpResource {
     });
   };
 
-  protected fromJSON(data: SignUpJSON | null): this {
+  protected;
+
+  fromJSON(data: SignUpJSON | null): this {
```

Comment:
:wrench: Is this change intentional? I believe the `protected` modifier should come right before the method definition.

Reply:
Messed up while auto-formatting, thanks for spotting